packages/databricks-vscode/src/extension.ts

```diff
@@ -24,15 +24,18 @@ import {showQuickStartOnFirstUse} from "./quickstart/QuickStart";
 import {PublicApi} from "@databricks/databricks-vscode-types";
 import {LoggerManager, Loggers} from "./logger";
 import {NamedLogger} from "@databricks/databricks-sdk/dist/logging";
-import {workspaceConfigs} from "./WorkspaceConfigs";
+import {workspaceConfigs} from "./vscode-objs/WorkspaceConfigs";
 import {PackageJsonUtils, UtilsCommands} from "./utils";
 import {ConfigureAutocomplete} from "./language/ConfigureAutocomplete";
 import {WorkspaceFsCommands, WorkspaceFsDataProvider} from "./workspace-fs";
 import {generateBundleSchema} from "./bundle/GenerateBundle";
+import {CustomWhenContext} from "./vscode-objs/CustomWhenContext";
 
 export async function activate(
     context: ExtensionContext
 ): Promise<PublicApi | undefined> {
+    CustomWhenContext.setActivated(false);
```

Comment:
shouldn't we move this line down after the version and workspace checks?

Reply:
This is to make sure that the extension context is set to deactivated when it fails to activate due to any reason. Which includes failure due to version and workspace checks.